demosplan/DemosPlanCoreBundle/Logic/Permission/AccessControlPermissionPerOrgaService.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Logic\Permission;
+
+use demosplan\DemosPlanCoreBundle\Entity\Permission\AccessControlPermission;
+use demosplan\DemosPlanCoreBundle\Logic\CoreService;
+use demosplan\DemosPlanCoreBundle\Logic\User\RoleHandler;
+use demosplan\DemosPlanCoreBundle\Repository\AccessControlPermissionRepository;
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+class AccessControlPermissionPerOrgaService extends CoreService
```

Comment:
Why did you build this specifically for Orgas?

Reply:
The big picture I could imagine is that the dynamic permissions can be applied at any application level like global, customer, organization, procedure (sometimes in the future perhaps even user) in combination with any (or specific or none) roles. Permissions could then be granted in the interface when and how needed. 
I think we will need a list of permissions, that are available for this dynamic mechanism. It probably would be best if you introduce it now from the beginning of this feature.

Reply:
Not sure whether we would want to migrate existing permissions into the database. I would rather stress test this new concept and decide then